app/test/socobo-inventory/socobo-inventory-behavior.html

```diff
@@ -32,15 +32,15 @@
       });
 
       test('valid items are validated correctly', function(done) {
-        var validItem = item.extend({name: "Valid name", count: "4", measurement: {__firebaseKey__: "0", value: "pcs"}});
+        var validItem = Object.assign(item, {name: "Valid name", amount: "4", unit: {__firebaseKey__: "0", value: "pcs"}});
```

Comment:
Yes, but it works in the current FF and Chrome and that is good enough for running the tests, or not?